stdlib/math.ysh

```diff
@@ -1,79 +1,79 @@
+func _select(list, cmp) {
```

Comment:
Hm I realized that this naming convention isn't YSH-like

Because we have public APIs like `_match()` and `_status` .  That convention came from PHP, where you have `_GET[]` and `_POST`.    I needed something that was easy to type but distinct from the user's variables

So we could maybe settle on `__select`, or even `__math_select` which is kind of Python style

( It is a bit annoying that we have a big global namespace, not sure how high a priority changing that is though )